datalad/distribution/get.py

```diff
@@ -1000,7 +1025,7 @@ def __call__(
                             continue
                         yield res
 
-        if not get_data:
+        if data=='nothing' or not get_data:
```

Comment:
I feel that introduction of `data` potentially makes `get_data` obsolete and thus require replacement/deprecation cycle and that could be taken as a good thing since `get_data` flag semantic is already shown to be "inconsistent" (`get_data` in python but `--no-data` in CLI).

But `-n` (AKA `--no-data`) is used heavily in handbook etc, and is a convenient shortcut to signal `--data=nothing`.

So, I think that is how `-n|--no-data` should be promoted - that it is a shortcut for `data="nothing"` and there should be explicit statement that `data` option is ignored if `--no-data` is provided.

But also let's wait for others to chime in